### PR TITLE
Combo PR: Added new about window and implemented minimum window size.

### PR DESCRIPTION
**Pull request by R4H33M**
Fixes: #82 and #96 

Sorry, I know I should actually make separate PRs - I forgot to create a new branch for the minimum window size commit. 
I hope it is still okay.

### CODE REVIEW

File: sources-experimental/MainWindow.cpp [hunk 1]
```diff
@@ -84,7 +84,8 @@ extern IActionManagerBP		action_manager;
 
 
 MainWindow::MainWindow()
-: BWindow(BRect(100,100,970,607), B_TRANSLATE_SYSTEM_NAME("BePodder"), B_TITLED_WINDOW, B_ASYNCHRONOUS_CONTROLS),
+: BWindow(BRect(100,100,970,607), B_TRANSLATE_SYSTEM_NAME("BePodder"), B_TITLED_WINDOW,
```
Comment (owenca) [inline sources-experimental/MainWindow.cpp]:
```suggestion
	:
	BWindow(BRect(100, 100, 970, 607), B_TRANSLATE_SYSTEM_NAME("BePodder"),
		B_TITLED_WINDOW, B_ASYNCHRONOUS_CONTROLS | B_AUTO_UPDATE_SIZE_LIMITS),
```

File: sources-experimental/MainWindow.cpp [hunk 2]
```diff
@@ -84,7 +84,8 @@ extern IActionManagerBP		action_manager;
...
+	B_ASYNCHRONOUS_CONTROLS | B_AUTO_UPDATE_SIZE_LIMITS),
```
Comment (owenca) [inline sources-experimental/MainWindow.cpp]:
```suggestion
```

**Comment by R4H33M** [comment]
Done.
I added an extra newline in order to stay within Haiku's 80 char line limit.

**Comment by korli** [comment]
Could you please squash the style fix commit in the first commit? 
Could you please amend the first commit message with the issue #96?
Could you please amend the second commit message with the issue #82?
When it's done, it's easier to rebase, merge and close related issues. Thanks.

**Comment by R4H33M** [comment]
Is this better? Rebasing is really funky, so I hope this is what you wanted me to do.

**Comment by owenca** [comment]
> I added an extra newline in order to stay within Haiku's 80 char line limit.

@R4H33M It fits 80 columns. If you are using an editor other than Pe and Kodar, make sure you set the tab stops to 4 columns.

**Pull request merged**